x/tools/imports/export.go

```diff
@@ -27,20 +34,57 @@ func GoPath() map[string]*Pkg {
 }
 
 func exportDirScan(ds map[string]*pkg) map[string]*Pkg {
+	var wg sync.WaitGroup
+	var rm sync.Mutex
 	r := make(map[string]*Pkg)
-	for path, pkg := range ds {
-		r[path] = exportPkg(pkg)
+
+	// async function
```

Comment:
I wonder is it really fast implementation.
It may be bad idea to spawn goroutine for each all packages. Shouldn't we need semaphore or something?

I think it's likely we don't need to make it async in the first place.
Do you measure time and performance?